Sources/BraintreeVenmo/Public/BraintreeVenmo/BTVenmoAccountNonce.h

```diff
@@ -9,6 +9,31 @@
  */
 @interface BTVenmoAccountNonce : BTPaymentMethodNonce
 
+/**
```

Comment:
Are they able to get these values back from both Venmo flows (singleUse and multiUse)? If not - we should denote which one.